### DISCUSSION TITLE
how can I get the number of essential bits of the generated PYNQ's bitfile?

### DISCUSSION THREAD

**Discussion by ArthurEly** [Xilinx/finn#1025] [Q&A] [answered]
Hi!! How u doing?

Is there a way to get this information in the FINN's flow? Right now, I'm struggling to find a way to get this metric on the Zynq project that is used to generate the bitfile.

I'll get the correct/same # of essential bits if I generate a bitstream for the Stitched IP project? 

Thank you!

**Comment by auphelia** [comment, marked as answer]
Hi @ArthurEly, we don't have a way to report this directly from the FINN flow. Please refer to the Vivado documentation regarding this.
